src/components/KCard/KCard.cy.ts

```diff
@@ -13,14 +13,16 @@ describe('KCard', () => {
 
   it('renders title prop when passed', () => {
     const titleProp = 'Title prop'
+    const titleTag = 'h5'
 
     mount(KCard, {
       props: {
         title: titleProp,
+        titleTag,
       },
     })
 
-    cy.get('.k-card').find('.card-title').should('contain', titleProp)
+    cy.get('.k-card').find(`${titleTag}.card-title`).should('contain', titleProp)
```

Comment:
since a prop is determining the HTML tag, I'd also add a test case in here to ensure that the prop.titleTag = the HTML tag element

Reply:
It is exactly what this test is doing: I'm setting `titleTag` prop and then checking whether correct element is rendered.. or did I miss something?

Reply:
Ah I missed the `${}` selector. Yep.